src/library.ts

```diff
@@ -1,3 +0,0 @@
-export default class DummyClass {
```

Comment:
Apparently you removed this folder and file. Please revert that, is there on purpose, otherwise you'll see running `npm t` would fail.